### ISSUE TITLE
Revisar a documentação do Potigol

### ISSUE THREAD

**Issue by lrlucena** [potigol/potigol.github.io#2]
Revisar o texto da documentação da Linguagem Potigol

**Comment by klisostom** [comment]
Revisar formatação ou possíveis erros de português professor?

**Comment by lrlucena** [comment]
Sim. Corrigir erros, melhorar o texto, apontar pontos fracos ...

**Comment by klisostom** [comment]
Ok. Vou fazer o fork e trabalhar nisso no meu tempo vago.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐
Em Quinta-feira, 3 de Outubro de 2019 às 17:15, Leonardo Lucena <notifications@github.com> escreveu:

**Comment by klisostom** [comment]
> Sim. Corrigir erros, melhorar o texto, apontar pontos fracos ...

O estilo de identação é de quantas linhas? 4?
O exemplo que me deixou confuso foi este:
```
y = para i de 1 até 10,
         j de i + 1 até 10 gere
           i+j
    fim
escreva y
```

**Comment by lrlucena** [comment]
O estilo padrão são 2 espaços. Neste exemplo a regra foi alinhar o 'fim' com o 'para' e o 'i' com o 'j'. Realmente parece confuso. Poderia ser tudo em uma linha só mas acho que ficaria muito extenso. Sugestões são bem vindas.
